common/models.py

```diff
@@ -25,7 +25,7 @@ def get_state_dictionary(self):
         return self.model.fc.state_dict()
 
     def load_last_layer_state_dictionary(self, state_dict):
-        self.model.fc.load_state_dict(state_dict)
+        self.model.fc.load_state_dict(torch.load(state_dict))
```

Comment:
don't arbitrarily change this method, instead do torch.load when you call load_last_layer_state_dict